src/platform_impl/windows/icon.rs

```diff
@@ -239,13 +233,26 @@ impl WinCursor {
                 return Err(io::Error::last_os_error());
             }
 
-            Ok(Self::from_handle(handle))
+            Ok(Self::Cursor(Arc::new(RaiiCursor { handle })))
+        }
+    }
+
+    pub(crate) fn build<T>(
+        cursor: OnlyCursorImageBuilder,
+        _: &EventLoopWindowTarget<T>,
+    ) -> WinCursor {
```

Comment:
minor: I would prefer replacing the instances of `WinCursor` with `Self`

Reply:
Done.